elearning-backend/routes/api.php

```diff
@@ -20,7 +21,14 @@
 Route::post('/register', [UserController::class, 'store']);
 
 Route::middleware('auth:sanctum')->group(function () {
+    // Authenticated Routes
     Route::post('/user', [UserProfileController::class, 'index']);
     Route::post('/update-user', [UserController::class, 'update']);
     Route::post('/logout', [SessionsController::class, 'logout']);
-});
\ No newline at end of file
+
+    // Admin Routes
+    Route::post('/admin/all-users', [AdminUsersController::class, 'index']);
+    Route::post('/admin/single-user', [AdminUsersController::class, 'getUser']);
+    Route::post('/admin/update-user', [AdminUsersController::class, 'update']);
+    Route::post('/admin/delete-user', [AdminUsersController::class, 'delete']);
```

Comment:
```suggestion
    Route::get('/admin/all-users', [AdminUsersController::class, 'index']);
    Route::get('/admin/single-user', [AdminUsersController::class, 'getUser']);
    Route::put('/admin/update-user', [AdminUsersController::class, 'update']);
    Route::delete('/admin/delete-user', [AdminUsersController::class, 'delete']);
```
All your method is incorrect, pls use proper method types
```
get -> fetching
put/patch -> updating
delete -> deleting
```